pydriller/domain/commit.py

```diff
@@ -562,6 +590,161 @@ def _get_branches(self):
             branches.add(branch.strip().replace('* ', ''))
         return branches
 
+
+    _UNIT_SIZE_LOW_RISK_THRESHOLD = 15
```

Comment:
Does this threshold come from the paper and/or SIG MM? 
Maybe we should add a link or something. I am expecting someone to ask "why 15?" 😄 
Similarly, check the other metrics.